app/assets/javascripts/edit_plan.js

```diff
@@ -100,10 +100,11 @@ $(document).on("ready ajax:success", function() {
     var total_premium_value = document.getElementById("enrollment_total_premium").innerHTML;
     var total_premium = toFixedTrunc(parseFloat(total_premium_value));
     // Max available tax credit per month for month
+    var max_aptc = document.getElementById("max_tax_credit");
     var max_aptc_available = document.getElementById("max_aptc_available").innerHTML;
-    var new_percent = tax_value/max_aptc_available;
+    var new_percent = max_aptc ? tax_value/max_aptc.innerHTML : tax_value/max_aptc_available;
 
-    var aptc_total_cash_amount_to_apply = toFixedTrunc(max_aptc_available * new_percent);
+    var aptc_total_cash_amount_to_apply = max_aptc ? toFixedTrunc(max_aptc.innerHTML * new_percent) : toFixedTrunc(max_aptc_available * new_percent);
```

Comment:
same comment as above. is there any case where max_aptc is not present.

Reply:
@nisanthyaganti9 max_aptc won't be present in the DC context